docs/buildah-bud.md

```diff
@@ -178,6 +178,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--decryption-key** *key*
+
+A reference required to perform decryption of container images. This should point to files which represent keys and/or certificates that can be used for decryption. Decryption will be tried with all keys. This feature is still *experimental*.
```

Comment:
“reference”?

Reply:
I think in the context of that it's because its pointing to a file and not the key data itself. This is made to be consistent with the skopeo docs.

Reply:
Huh, so it is consistent with Skopeo. I haven’t noticed :)

Basically I think “A reference required to perform decryption of container images. This should point to files which represent… that can be used for decryption” is just a very long way to say “Path to a file containing… required for decryption” or something like that. 

I don’t see any additional value gained by adding “reference”, “point to”, “represent”.

Reply:
Yea.. i think the way this got constructed was because the way encryption-key is specified is that it requires `<protocol>:<file>`... But I do agree, would be a good opportunity to reword this. I can update this and then once this gets merged ill update skopeo docs.